scripts/ci/ci_refresh_pylint_todo.sh

```diff
@@ -23,13 +23,21 @@ MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 export AIRFLOW_CI_SILENT=${AIRFLOW_CI_SILENT:="false"}
 export FORCE_ANSWER_TO_QUESTIONS=quit
 
-# shellcheck source=scripts/ci/_utils.sh
-. "${MY_DIR}/_utils.sh"
-
-basic_sanity_checks
+# shellcheck source=scripts/ci/utils/_init.sh
+. "${MY_DIR}/utils/_init.sh"
+# shellcheck source=scripts/ci/utils/_build.sh
+. "${MY_DIR}/utils/_build.sh"
+# shellcheck source=scripts/ci/utils/_run.sh
+. "${MY_DIR}/utils/_run.sh"
```

Comment:
These lines seem to be repeated a lot. Is there any way of merging this into a single command and sourcing it?

Reply:
Sure :). Will do.